src/plugins/addLoadingFlag.js

```diff
@@ -2,12 +2,13 @@ import { wrap, Chart } from 'highcharts/highstock';
 
 export default () => {
     wrap(Chart.prototype, 'showLoading', function showLoading(proceed, ...args) {
-        this.isLoading = true;
+        if (Object.keys(this).length === 0) return;
```

Comment:
Is that the best way to check?
Is this what you meant typeof this === 'undefined'
If so, why does it happen?